package.json

```
@@ -125,12 +125,28 @@
                 "title": "Motoko: Restart language server"
             },
             {
-                "command": "motoko.deployPlayground",
+                "command": "motoko.deployPlaygroundSelection",
                 "title": "Motoko: Deploy (20 minutes)"
             },
             {
                 "command": "motoko.importMopsPackage",
                 "title": "Motoko: Import Mops Package..."
+            },
+            {
+                "command": "motoko.startReplica",
+                "title": "Motoko: Start local replica..."
+            },
+            {
+                "command": "motoko.deployLocalSelection",
+                "title": "Motoko: Deploy (local node)..."
+            },
+            {
+                "command": "motoko.candidLocalSelection",
+                "title": "Motoko: Open Candid UI for canister (local node)..."
+            },
+            {
+                "command": "motoko.candidPlaygroundSelection",
+                "title": "Motoko: Open Candid UI for canister (playground)..."
```

For context, the `...` suffix is used in this extension to indicate that you'll see a dropdown menu after selecting this command. Here are some suggested changes to the command titles:

```suggestion
                "title": "Motoko: Start local replica"
            },
            {
                "command": "motoko.deployLocalSelection",
                "title": "Motoko: Deploy (local replica)..."
            },
            {
                "command": "motoko.candidLocalSelection",
                "title": "Motoko: Open Candid UI for canister (local replica)..."
            },
            {
                "command": "motoko.candidPlaygroundSelection",
                "title": "Motoko: Open Candid UI for canister (playground)..."
```